packages/stardog-language-utils/src/__tests__/namespaceUtils.test.ts

```diff
@@ -82,12 +82,21 @@ describe('abbreviatePrefixArray and abbreviatePrefixObj', () => {
     testArrayAndObj('http://foo.org/bar/fizz/buzz', namespaces, 'fizz:buzz');
```

Comment:
@anneeb do you know what's up with ci here? we haven't touched this repo in a while :)